openshift/dynamic/client.py

```diff
@@ -916,7 +987,7 @@ def main():
         item = {}
         item[key] = {k: v for k, v in resource.__dict__.items() if k not in ('client', 'subresources', 'resource')}
         if isinstance(resource, ResourceList):
-            item[key]["resource"] = '{}.{}'.format(resource.resource.group_version, resource.resource.kind)
+            item[key]["resource"] = '{}.{}'.format(resource.group_version, resource.kind)
```

Comment:
Should this be `resource.base_kind` instead of `resource.kind`?
Maybe you don't need this at all because base_kind is already an attribute of the resource.